dask/dataframe/io/parquet/arrow.py

```diff
@@ -335,9 +335,15 @@ def read_metadata(
         ignore_metadata_file=False,
         metadata_task_size=0,
         parquet_file_extension=None,
+        storage_options=None,
         **kwargs,
     ):
 
+        # Set default open_file_options for remote filesystems
+        kwargs["open_file_options"] = cls._default_open_file_options(
```

Comment:
> `_get_open_file_options`

Maybe this is dumb, but rather than build this convention, could we do something like ...

```python
if engine == "pyarrow" and filename.startswith("s3://") and not storage_options:
    open_file_options[...] = ...
```

Building a convention like `_get_open_file_options` for this seems a bit heavyweight to me.

(please feel free to entirely ignore this comment.  I also may not ever respond.  Please do not block on me)